stubs/Stringable.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+if (\PHP_VERSION_ID < 80000 && !interface_exists('Stringable')) {
```

Comment:
`symfony/polyfill-php80` has no if condition there: https://github.com/symfony/polyfill-php80/blob/main/Resources/stubs/Stringable.php
Is there a way to control what would composer use in the classmap for this interface as a file? I guess it should be only 1 file and it doesn't matter which one of them will get here if both would be available.
Do you see a issue where symfony might be included after this one and having no if condition it would fail?

Reply:
Oh good point 🤔

Reply:
It doesn't matter which one is used as both polyfill "emulate" the same interface.

The classmap autoloads will result in an associative array which associate a class with a file which implement this one.
From the Composer autoload view, only one file will be loaded.